lib/config/presets/util.ts

```diff
@@ -35,6 +35,9 @@ export async function fetchPreset({
         throw err;
       }
       logger.debug('default.json preset not found - trying renovate.json');
+      logger.info(
+        'Using a renovate.json file as a preset is deprecated, please use a default.json file instead.'
+      );
```

Comment:
I don't think the new `logger.info` message is actually triggering correctly. I think I need to move it down a bit?

Reply:
i think info is ok for now, but message is a little bit misleading. Maybe this ?
```suggestion
      logger.info(
        'Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead.'
      );
```